evalml/tests/component_tests/test_time_series_featurizer.py

```diff
@@ -252,7 +252,7 @@ def test_delayed_feature_extractor_maxdelay5_forecasthorizon1_gap0(
     f"evalml.pipelines.TimeSeriesFeaturizer.{ROLLING_TRANSFORM_METHOD_NAME}",
     return_value=pd.DataFrame(),
 )
-def test_delayed_feature_extractor_maxdelay3_forecasthorizon7_gap1(
+def test_delayed_feature_extractor_maxdelay2_forecasthorizon7_gap1(
```

Comment:
What was the reasoning behind changing this test?

Reply:
IIRC just to match the lags on the dataset